Entrega/exercicio_3.js

```diff
@@ -0,0 +1,12 @@
+var fishWeight = Number(prompt("Digite o peso de peixes"))
+
+var excess = fishWeight - 50
+
+var penalty = excess * 4;
+
+if (fishWeight <=50) {
+    console.log ("Excesso: 0, Multa: R$ 0,00")
+}
+else if (fishWeight >=51){
```

Comment:
não está errado, mas vc pode usar só o `else` pq se ele for 51, por exemplo, vai entrar ai de qualquer jeito :D